### DISCUSSION TITLE
Installing rdkit does not install FreeSASA

### DISCUSSION THREAD

**Discussion by Acario** [rdkit/rdkit#4572] [Q&A]
I have started a project recently where I need to find the topological surface area of molecules. I have been using rdkit for many of my cheminformatics needs and figured I would use rdkit for this purpose as well. However, there is no FreeSASA functions as described.

https://www.rdkit.org/docs/source/rdkit.Chem.rdFreeSASA.html

I looked into the rdkit files on my computer to find there is no FreeSASA file. Is there a reason why this is the case? Am I missing something?

I have a windows 10 64-bit OS and use Anaconda 2021.05 which installs python 3.8.8. rdkit 2021.03.5 is currently installed. When looking into possible update issues, I did find one with Anaconda Navigator not being the most recent update. This issue was resolved and now my computer shows I am up to date on all the applications. It is my understanding that rdkit 2021.03.5 is the most current version Anaconda users are able to install. Also since FreeSASA was implemented into rdkit about 3 years ago, I do not think the slightly older version Anaconda installs is the reason for the issue. 

When speaking to my coworker about it, he also tried to resolve and got to the same issue I am having. He seems to think it has something to do with the .h file of FreeSASA which prevents the installation.

 I did a search on this github and have not found any previous discussion or bug report similar to my issue, but if I missed it, please direct me to a solution. If you need any further information from me to help find a solution, let me know.

**Comment by manajit-das** [comment]
Hi Acario, check the answer suggested by ptosco on my question. It may help. https://github.com/rdkit/rdkit/discussions/4579

**Reply by greglandrum** [reply]
FreeSASA is unfortunately not supported on Windows.
